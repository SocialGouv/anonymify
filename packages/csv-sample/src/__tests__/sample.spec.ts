import fs from "fs";

import { sample } from "../index";

test("should analyse sample.csv", async () => {
  const readStream = fs.createReadStream(`./sample.csv`);

  const samples = await sample(readStream, { onProgress: console.log });

  expect(samples).toMatchSnapshot();
});
