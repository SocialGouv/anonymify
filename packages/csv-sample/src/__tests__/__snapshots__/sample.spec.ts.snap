// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should analyse sample.csv 1`] = `
[
  {
    "name": "civilite",
    "type": "sexe",
    "values": [
      "Homme",
      "Femme",
    ],
  },
  {
    "name": "column_9",
    "type": "ville",
    "values": [
      "MANTES LA JOLIE",
      "AUBERGENVILLE",
      "PORCHEVILLE",
      "POISSY",
      "CONFLANS STE HONORINE",
      "HARDRICOURT",
      "TRIEL SUR SEINE",
      "MEULAN",
      "ANDRESY",
      "ORGEVAL",
      "ACHERES",
      "CARRIERES SOUS POISSY",
      "LES MUREAUX",
      "CONFLANS ST HONORINE",
      "GARGENVILLE",
      "ROSNY SUR SEINE",
      "VERNEUIL SUR SEINE",
    ],
  },
  {
    "name": "column_10",
    "type": "tel",
    "values": [
      "01.30.33.39.82",
      "01.30.95.12.20",
      "01.30.33.14.61",
      "01.30.65.87.87",
      "01.39.72.66.57",
      "01.30.95.47.14",
      "01.39.65.15.49",
      "01.30.91.99.34",
      "01.39.75.18.61",
      "01.30.91.46.04",
      "01.39.74.51.68",
      "01.39.75.88.20",
      "01.39.11.35.45",
      "01.39.65.38.66",
      "01.39.74.26.36",
      "01.34.74.20.63",
      "01.39.74.17.38",
      "01.39.72.61.19",
      "01.30.93.50.88",
      "01.30.42.80.93",
      "01.39.70.70.70",
      "01.39.65.66.70",
      "01.30.95.81.86",
    ],
  },
  {
    "name": "column_11",
    "type": "integer",
    "values": [
      "18",
      "12",
      "67",
    ],
  },
  {
    "name": "column_12",
    "type": "empty",
    "values": [],
  },
  {
    "name": "column_13",
    "type": "text",
    "values": [
      "Libéral intégral",
      "Libéral activité salariée",
      "Libéral temps partiel hospitalier",
    ],
  },
  {
    "name": "column_14",
    "type": "text",
    "values": [
      "Secteur 1 ou conventionné",
      "Secteur 2, Pas de contrat d'accès aux soins",
    ],
  },
  {
    "name": "column_15",
    "type": "fixed",
    "values": [
      "N",
    ],
  },
  {
    "name": "column_16",
    "type": "fixed",
    "values": [
      "Lecteur de carte Sesam Vitale",
    ],
  },
  {
    "name": "column_17",
    "type": "text",
    "values": [
      "HBLD2270",
      "MEMC0050",
      "HBFD0330",
      "HBLD0310",
      "HBMD0470",
      "JHQM0010",
      "LAQJ0010",
      "MDQK0020",
      "MZQK0020",
      "HBGD0430",
      "HBLD4350",
      "NZQJ0010",
      "PBJH0010",
      "ZCQK0040",
      "HBBD0040",
      "HBMD0200",
      "HBMD0380",
      "HBQK3890",
      "HBMD0170",
      "HBGD0220",
      "HBBD0060",
      "HBFD0210",
      "HBLD4030",
      "HBMD0530",
      "HBMD2920",
      "HBGD0350",
      "HBMD0500",
      "HBGD0360",
      "HBMD3510",
      "HBLD4520",
      "HBLD6340",
      "HBLD2450",
      "HBFD0030",
      "HBQK0020",
      "HBLD4180",
      "HBGD0370",
      "HBGD0160",
      "HBLD0900",
      "HBMD0490",
    ],
  },
  {
    "name": "column_18",
    "type": "integer",
    "values": [
      "158",
      "60",
      "87",
      "157",
      "212",
      "100",
      "142",
      "177",
      "185",
      "128",
      "140",
      "188",
      "163",
      "183",
      "211",
      "167",
      "7",
    ],
  },
  {
    "name": "column_19",
    "type": "float",
    "values": [
      "1073.5",
      "50.2",
      "86.8",
      "61.0",
      "8.0",
      "60.0",
      "27.0",
      "45.4",
      "33.4",
      "450.0",
      "500.0",
      "390.0",
      "20.0",
      "175.0",
    ],
  },
  {
    "name": "column_20",
    "type": "float",
    "values": [
      "873.5",
      "50.0",
      "400.0",
      "500.0",
      "175.0",
    ],
  },
  {
    "name": "column_21",
    "type": "float",
    "values": [
      "1073.5",
      "85.0",
      "450.0",
      "530.0",
      "580.0",
      "230.0",
    ],
  },
  {
    "name": "column_22",
    "type": "empty",
    "values": [],
  },
  {
    "name": "column_23",
    "type": "empty",
    "values": [],
  },
  {
    "name": "column_24",
    "type": "empty",
    "values": [],
  },
  {
    "name": "column_25",
    "type": "empty",
    "values": [],
  },
  {
    "name": "column_26",
    "type": "empty",
    "values": [],
  },
  {
    "name": "column_27",
    "type": "empty",
    "values": [],
  },
  {
    "name": "column_28",
    "type": "empty",
    "values": [],
  },
  {
    "name": "column_29",
    "type": "empty",
    "values": [],
  },
  {
    "name": "column_30",
    "type": "empty",
    "values": [],
  },
  {
    "name": "column_31",
    "type": "empty",
    "values": [],
  },
  {
    "name": "column_32",
    "type": "empty",
    "values": [],
  },
  {
    "name": "column_33",
    "type": "empty",
    "values": [],
  },
  {
    "name": "nom",
    "type": "text",
    "values": [
      "JACQUES LEFEVRE",
      "PHILIPPE ROURE",
      "ANNA OLEJNIK BAILLY",
      "DOMINIQUE CARIO",
      "FRANCOIS AUBERT",
      "JULIEN HERICOURT",
      "VIRGINIE GERARD",
      "ALAIN BENESTI",
      "JACQUES NGUYEN VAN NHIEU",
      "EVELYNE BROCHET LE FLAHEC",
      "ALAIN BEHOT",
      "ERIK AZUELOS",
      "ILANIT KRIEF",
      "OLIVIER MIZRAHI",
      "GILLES NAUDO",
      "AMELIE PLAT",
      "KARINA TEBEKA MIZRAHI",
      "YOHANE ELFASSI",
      "JEAN PIERRE FISCHER",
      "MICHAEL BENZAKKI",
      "PAUL JABIN",
      "JEAN BAPTISTE HAURET",
      "IMAN LAZIZ",
      "ELENI TSAGKARI",
      "GABRIEL ELLOUK",
      "BARBARA RICHARD",
      "SHANA ELLOUK",
      "LENA SIMONIN",
      "THOMAS GANDIL",
    ],
  },
  {
    "name": "adresse",
    "type": "text",
    "values": [
      "CABINET DU DR LEFEVRE  105 RUE MAURICE BRAUNSTEIN  78200 MANTES LA JOLIE",
      "SELARL PHILIPPE ROURE CHP DU MONTGARDE 32 RUE DE MONTGARDE  78410 AUBERGENVILLE",
      "CABINET DU DR OLEJNIK BAILLY  55 GRANDE RUE  78440 PORCHEVILLE",
      "CABINET DU DR CARIO SCM DRS CARIO ET LEGOIS 7 AVENUE DES URSULINES  78300 POISSY",
      "CTRE HOSPITALIER DU MONTGARDE HOP PRIVE DU MONTGARDE 32 RUE DU MONTGARDE  78410 AUBERGENVILLE",
      "CABINET DU DR HERICOURT  39 AVENUE MARECHAL GALLIENI  78700 CONFLANS STE HONORINE",
      "CABINET DENTAIRE  10 PLA J MONNET  78410 AUBERGENVILLE",
      "CABINET DU DR BENESTI  107 RUE DE VILLIERS  78300 POISSY",
      "CABINET DU DR NGUYEN VAN NHIEU RES DE LA GARE 2 AVENUE DE LA GARE  78250 HARDRICOURT",
      "32 RUE DU LT LECOMTE  78510 TRIEL SUR SEINE",
      "CABINET DU DR BEHOT  1 PLACE DU VEXIN  78250 MEULAN",
      "SELARL CABINET DENTAIRE DU DR E AZUELO  39 AVENUE DU MARECHAL FOCH  78570 ANDRESY",
      "CABINET DU DR KRIEF CABINET DU DR KRIEF 469 AVENUE DU MARECHAL FOCH  78630 ORGEVAL",
      "SELARL DE CHIRURGIENS DENTISTES KOOK SELARL CD KOOK 3 PLACE LOUISE GERMAIN POURLIER  78260 ACHERES",
      "CABINET DU DR NAUDO  45 RUE DES FLEURS  78955 CARRIERES SOUS POISSY",
      "SELARL CABINET DENTAIRE DU DR A MOUSSA SELARL CD DR A MOUSSAI 36 RUE DU MARECHAL FOCH  78570 ANDRESY",
      "SELARL DE CHIRURGIENS DENTISTES KOOK SELARL CHIR DENT KOOK 3 PLACE LOUISE GERMAIN POURLIER  78260 ACHERES",
      "SELARL DE CHIR DENT LES ETOILES SELARL CD LES ETOILES 24 RUE DES BOUGIMONTS  78130 LES MUREAUX",
      "CABINET DENTAIRE DU DR J P FISCHER SELARL 139 AVENUE DE L HAUTIL  78955 CARRIERES SOUS POISSY",
      "SELARL ELLOUK SELARL ELLOUK 58 RUE MAURICE BERTEAUX  78700 CONFLANS ST HONORINE",
      "SELARL MGM JABIN HAURET SELARL MGM JABIN HAURET 98 AVENUE DE PARIS  78440 GARGENVILLE",
      "SELARL ILBW SELARL ILBW 20 RUE NATIONALE  78710 ROSNY SUR SEINE",
      "SELARL ELLOUK DE CH DENT SELARL ELLOUK DE CH DENT 58 RUE MAURICE BERTAUX  78700 CONFLANS STE HONORINE",
      "SELARL LE FIL D ARIANE SELARL LE FIL D ARIANE 24 RUE DES BOUGIMONTS  78130 LES MUREAUX",
      "SELARL ELLOUK DE CHIRURGIEN DENTISTE SELARL ELLOUK CHIR DENT 58 RUE MAURICE BERTEAUX  78700 CONFLANS STE HONORINE",
      "SELARL DENTADDICT SELARL DENTADDICT 41 BIS GRANDE RUE  78480 VERNEUIL SUR SEINE",
      "CABINET DU DR GANDIL CAB DR GANDIL ET COULAUD 32 BOULEVARD DE MANTES  78410 AUBERGENVILLE",
    ],
  },
  {
    "name": "libelle_profession",
    "type": "text",
    "values": [
      "Chirurgien-dentiste",
      "Chirurgien orthopédiste et traumatologue",
      "Radiologue",
    ],
  },
  {
    "name": "type_dacte_realise",
    "type": "fixed",
    "values": [
      "Acte technique",
    ],
  },
  {
    "name": "nom_acte",
    "type": "text",
    "values": [
      "Pose de prothèse fixée (couronne, bridge)",
      "Chirurgie des os et de l'articulation de l'épaule (hors fracture - luxation),",
      "Dévitalisation de dent,",
      "Pose de prothèse amovible (appareil dentaire),",
      "Traitement des caries/fractures,",
      "Échographie de l'appareil urinaire, de l'appareil génital et du petit bassin",
      "IRM du crâne, de la face ou du cou",
      "Radiographie osseuse et articulaire du membre supérieur,",
      "Scanner des membres ou de la colonne vertébrale,",
      "Extraction de dents,",
      "IRM des membres ou de la colonne vertébrale,",
      "Scanner du thorax, du sein ou de l'abdomen",
      "Prévention dentaire chez l'enfant,",
      "Réparation de prothèse amovible (appareil dentaire),",
      "Traitement des caries,",
      "Radiographie dentaire,",
      "Acte préprothétique,",
    ],
  },
  {
    "name": "coordonnees",
    "type": "geo",
    "values": [
      "48.999359,1.703095",
      "48.957417,1.847309",
      "48.978486,1.783762",
      "48.928796,2.042797",
      "48.994178,2.077324",
      "48.959517,1.855129",
      "48.921473,2.021946",
      "49.005645,1.903975",
      "48.977413,2.012281",
      "49.004713,1.907034",
      "48.988117,2.064557",
      "48.921212,1.975355",
      "48.959872,2.067715",
      "48.942388,2.025684",
      "48.98743,2.064461",
      "48.988058,1.914995",
      "48.945357,2.035986",
      "48.993154,2.094306",
      "48.988224,1.814472",
      "49.000973,1.631072",
      "48.987614,1.914142",
      "48.992651,2.092745",
      "48.978617,1.984917",
      "48.970775,1.842129",
    ],
  },
  {
    "name": "commune",
    "type": "ville",
    "values": [
      "Mantes-la-Jolie",
      "Aubergenville",
      "Porcheville",
      "Poissy",
      "Conflans-Sainte-Honorine",
      "Hardricourt",
      "Triel-sur-Seine",
      "Meulan-en-Yvelines",
      "Andrésy",
      "Orgeval",
      "Achères",
      "Carrières-sous-Poissy",
      "Les Mureaux",
      "Gargenville",
      "Rosny-sur-Seine",
      "Verneuil-sur-Seine",
    ],
  },
  {
    "name": "code_insee",
    "type": "cp",
    "values": [
      "78361",
      "78029",
      "78501",
      "78498",
      "78172",
      "78299",
      "78624",
      "78401",
      "78015",
      "78466",
      "78005",
      "78123",
      "78440",
      "78267",
      "78531",
      "78642",
    ],
  },
  {
    "name": "nom_epci",
    "type": "fixed",
    "values": [
      "CU Grand Paris Seine et Oise",
    ],
  },
  {
    "name": "nom_dep",
    "type": "fixed",
    "values": [
      "YVELINES",
    ],
  },
  {
    "name": "nom_reg",
    "type": "fixed",
    "values": [
      "ILE-DE-FRANCE",
    ],
  },
  {
    "name": "code_epci",
    "type": "fixed",
    "values": [
      "200059889",
    ],
  },
  {
    "name": "insee_reg",
    "type": "fixed",
    "values": [
      "11",
    ],
  },
  {
    "name": "insee_dep",
    "type": "fixed",
    "values": [
      "78",
    ],
  },
  {
    "name": "activite_principale",
    "type": "fixed",
    "values": [
      "1",
    ],
  },
  {
    "name": "libelle_regroupement",
    "type": "text",
    "values": [
      "None",
      "Actes de chirurgie",
      "Actes d'endodontie",
      "Soins Dentaires",
      "Acte d'échographie",
      "Actes d'imagerie",
      "Actes techniques médicaux (hors imagerie)",
      "Actes de Prophylaxie et Prévention",
      "Prothèses Fixes Céramiques",
      "Actes Inlay-cores",
      "Pose d'implants ou de matériel pour implantologie",
    ],
  },
  {
    "name": "libelle",
    "type": "text",
    "values": [
      "Pose d'une prothèse plurale [bridge] comportant 2 piliers d'ancrage céramométalliques et 1 élément intermédiaire céramométallique pour le remplacement d'une dent autre qu'une incisive",
      "Acromioplastie sans prothèse avec arthroplastie acromioclaviculaire par résection de l'extrémité latérale de la clavicule, par arthroscopie",
      "Exérèse de la pulpe vivante d’une incisive ou d’une canine permanente",
      "Pose d'une prothèse amovible définitive complète unimaxillaire à plaque base résine",
      "Restauration d'une dent d’un secteur incisivocanin sur 2 angles par matériau inséré en phase plastique, sans ancrage radiculaire",
      "Échographie transcutanée unilatérale ou bilatérale du contenu des bourses",
      "Remnographie [IRM] de la face, avec injection intraveineuse de produit de contraste",
      "Radiographie bilatérale de la main et/ou du poignet, selon 1 incidence sur un seul cliché de face",
      "Scanographie unilatérale ou bilatérale de segment du membre supérieur, sans injection de produit de contraste",
      "Avulsion de 2 dents permanentes sur arcade sans alvéolectomie",
      "Pose d'une prothèse amovible définitive à châssis métallique, comportant 9 dents",
      "Remnographie [IRM] unilatérale ou bilatérale de segment du membre inférieur, avec injection de produit de contraste",
      "Évacuation de calcification articulaire d'un membre, par voie transcutanée avec guidage radiologique",
      "Scanographie de l'abdomen et du petit bassin [pelvis], sans injection intraveineuse de produit de contraste",
      "Comblement [Scellement] prophylactique des puits, sillons et fissures sur 4 dents",
      "Réparation d'une prothèse dentaire amovible en résine sans renfort métallique, fêlée ou fracturée",
      "Restauration d’une dent d’un secteur prémolomolaire sur 3 faces ou plus par matériau inséré en phase plastique sans ancrage radiculaire",
      "Radiographie intrabuccale rétroalvéolaire et/ou rétrocoronaire d'un secteur de 1 à 3 dents contigües",
      "Adjonction ou changement d'1 élément d'une prothèse dentaire amovible",
      "Avulsion d'1 dent permanente sur arcade avec alvéolectomie",
      "Comblement [Scellement] prophylactique des puits, sillons et fissures sur 2 dents",
      "Exérèse de la pulpe vivante d’une première prémolaire maxillaire",
      "Pose d'une couronne dentaire dentoportée céramocéramique",
      "Restauration d’une dent d’un secteur prémolomolaire sur 1 face par matériau inséré en phase plastique, sans ancrage radiculaire",
      "Adjonction ou changement de 2 éléments soudés d'une prothèse dentaire amovible sur châssis métallique",
      "Avulsion d'1 dent temporaire sur arcade",
      "Restauration d’une dent d’un secteur incisivocanin sur 2 faces par matériau inséré en phase plastique sans ancrage radiculaire",
      "Avulsion d'1 dent permanente sur arcade sans alvéolectomie",
      "Restauration d’une dent sur 2 faces ou plus par matériau incrusté [inlay-onlay] composite ou en alliage non précieux",
      "Pose d'une prothèse amovible définitive à châssis métallique, comportant 5 dents",
      "Pose d'une couronne dentaire dentoportée céramométallique sur une incisive, une canine ou une première prémolaire",
      "Pose d'une infrastructure coronoradiculaire [Inlay core] sous une couronne ou un pilier de bridge dentoportés à tarif libre",
      "Exérèse du contenu canalaire non vivant d’une prémolaire autre que la première prémolaire maxillaire",
      "Radiographie panoramique dentomaxillaire",
      "Pose d'une couronne dentaire implantoportée",
      "Avulsion de 2 dents temporaires sur arcade",
      "Avulsion d'1 racine incluse",
      "Pose d'une infrastructure coronoradiculaire [Inlay core] sous une couronne ou un pilier de bridge dentoportés sans reste à charge",
      "Restauration d’une dent d’un secteur prémolomolaire sur 2 faces par matériau inséré en phase plastique sans ancrage radiculaire",
    ],
  },
  {
    "name": "tarif_2",
    "type": "float",
    "values": [
      "279.5",
      "350.71",
      "40.0",
      "182.75",
      "94.6",
      "37.05",
      "69.0",
      "19.95",
      "25.27",
      "50.16",
      "258.0",
      "55.0",
      "41.8",
      "50.54",
      "86.76",
      "21.5",
      "63.6",
      "7.98",
      "33.44",
      "43.38",
      "61.0",
      "107.5",
      "27.6",
      "86.0",
      "25.0",
      "46.5",
      "100.0",
      "215.0",
      "120.0",
      "90.0",
      "68.0",
      "20.0",
      "30.0",
      "83.6",
    ],
  },
  {
    "name": "tarif_1",
    "type": "float",
    "values": [
      "279.5",
      "392.79",
      "40",
      "182.75",
      "94.6",
      "37.05",
      "69",
      "19.95",
      "25.27",
      "50.16",
      "258",
      "55",
      "63.65",
      "50.54",
      "86.76",
      "21.5",
      "63.6",
      "7.98",
      "33.44",
      "43.38",
      "61",
      "107.5",
      "27.6",
      "86",
      "25",
      "46.5",
      "100",
      "215",
      "120",
      "90",
      "68",
      "20",
      "30",
      "83.6",
    ],
  },
  {
    "name": "tarif_base_de_remboursement_securite_sociale",
    "type": "float",
    "values": [
      "279.5",
      "350.71",
      "40.0",
      "182.75",
      "94.6",
      "37.05",
      "69.0",
      "19.95",
      "25.27",
      "50.16",
      "258.0",
      "55.0",
      "63.65",
      "50.54",
      "86.76",
      "21.5",
      "63.6",
      "7.98",
      "33.44",
      "43.38",
      "61.0",
      "107.5",
      "27.6",
      "86.0",
      "25.0",
      "46.5",
      "100.0",
      "215.0",
      "120.0",
      "90.0",
      "68.0",
      "20.0",
      "30.0",
      "83.6",
    ],
  },
  {
    "name": "libelle_acte_clinique",
    "type": "empty",
    "values": [],
  },
  {
    "name": "concat",
    "type": "text",
    "values": [
      "JACQUES LEFEVRECABINET DU DR LEFEVRE  105 RUE MAURICE BRAUNSTEIN  78200 MANTES LA JOLIE",
      "PHILIPPE ROURESELARL PHILIPPE ROURE CHP DU MONTGARDE 32 RUE DE MONTGARDE  78410 AUBERGENVILLE",
      "ANNA OLEJNIK BAILLYCABINET DU DR OLEJNIK BAILLY  55 GRANDE RUE  78440 PORCHEVILLE",
      "DOMINIQUE CARIOCABINET DU DR CARIO SCM DRS CARIO ET LEGOIS 7 AVENUE DES URSULINES  78300 POISSY",
      "FRANCOIS AUBERTCTRE HOSPITALIER DU MONTGARDE HOP PRIVE DU MONTGARDE 32 RUE DU MONTGARDE  78410 AUBERGENVILLE",
      "JULIEN HERICOURTCABINET DU DR HERICOURT  39 AVENUE MARECHAL GALLIENI  78700 CONFLANS STE HONORINE",
      "VIRGINIE GERARDCABINET DENTAIRE  10 PLA J MONNET  78410 AUBERGENVILLE",
      "ALAIN BENESTICABINET DU DR BENESTI  107 RUE DE VILLIERS  78300 POISSY",
      "JACQUES NGUYEN VAN NHIEUCABINET DU DR NGUYEN VAN NHIEU RES DE LA GARE 2 AVENUE DE LA GARE  78250 HARDRICOURT",
      "EVELYNE BROCHET LE FLAHEC  32 RUE DU LT LECOMTE  78510 TRIEL SUR SEINE",
      "ALAIN BEHOTCABINET DU DR BEHOT  1 PLACE DU VEXIN  78250 MEULAN",
      "ERIK AZUELOSSELARL CABINET DENTAIRE DU DR E AZUELO  39 AVENUE DU MARECHAL FOCH  78570 ANDRESY",
      "ILANIT KRIEFCABINET DU DR KRIEF CABINET DU DR KRIEF 469 AVENUE DU MARECHAL FOCH  78630 ORGEVAL",
      "OLIVIER MIZRAHISELARL DE CHIRURGIENS DENTISTES KOOK SELARL CD KOOK 3 PLACE LOUISE GERMAIN POURLIER  78260 ACHERES",
      "GILLES NAUDOCABINET DU DR NAUDO  45 RUE DES FLEURS  78955 CARRIERES SOUS POISSY",
      "AMELIE PLATSELARL CABINET DENTAIRE DU DR A MOUSSA SELARL CD DR A MOUSSAI 36 RUE DU MARECHAL FOCH  78570 ANDRESY",
      "KARINA TEBEKA MIZRAHISELARL DE CHIRURGIENS DENTISTES KOOK SELARL CHIR DENT KOOK 3 PLACE LOUISE GERMAIN POURLIER  78260 ACHERES",
      "YOHANE ELFASSISELARL DE CHIR DENT LES ETOILES SELARL CD LES ETOILES 24 RUE DES BOUGIMONTS  78130 LES MUREAUX",
      "JEAN PIERRE FISCHERCABINET DENTAIRE DU DR J P FISCHER SELARL 139 AVENUE DE L HAUTIL  78955 CARRIERES SOUS POISSY",
      "MICHAEL BENZAKKISELARL ELLOUK SELARL ELLOUK 58 RUE MAURICE BERTEAUX  78700 CONFLANS ST HONORINE",
      "PAUL JABINSELARL MGM JABIN HAURET SELARL MGM JABIN HAURET 98 AVENUE DE PARIS  78440 GARGENVILLE",
      "JEAN BAPTISTE HAURETSELARL MGM JABIN HAURET SELARL MGM JABIN HAURET 98 AVENUE DE PARIS  78440 GARGENVILLE",
      "IMAN LAZIZSELARL ILBW SELARL ILBW 20 RUE NATIONALE  78710 ROSNY SUR SEINE",
      "ELENI TSAGKARISELARL CABINET DENTAIRE DU DR E AZUELO  39 AVENUE DU MARECHAL FOCH  78570 ANDRESY",
      "GABRIEL ELLOUKSELARL ELLOUK DE CH DENT SELARL ELLOUK DE CH DENT 58 RUE MAURICE BERTAUX  78700 CONFLANS STE HONORINE",
      "BARBARA RICHARDSELARL LE FIL D ARIANE SELARL LE FIL D ARIANE 24 RUE DES BOUGIMONTS  78130 LES MUREAUX",
      "SHANA ELLOUKSELARL ELLOUK DE CHIRURGIEN DENTISTE SELARL ELLOUK CHIR DENT 58 RUE MAURICE BERTEAUX  78700 CONFLANS STE HONORINE",
      "LENA SIMONINSELARL DENTADDICT SELARL DENTADDICT 41 BIS GRANDE RUE  78480 VERNEUIL SUR SEINE",
      "THOMAS GANDILCABINET DU DR GANDIL CAB DR GANDIL ET COULAUD 32 BOULEVARD DE MANTES  78410 AUBERGENVILLE",
    ],
  },
]
`;
